### PR TITLE
LPD-1872 Types

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/contexts/PortletContentContext.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+						segmentsExperienceId: segmentsExperienceIdRef.current!,
+					}).then(({content}) => {
+						dispatch(
+							updateFragmentEntryLinkContent({
+								content,
+								fragmentEntryLinkId,
+							})
+						);
+
+						removePendingItem(pendingItemsRef, fragmentEntryLinkId);
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/contexts/PortletContentContext.tsx:64]: :thinking: you are right, I'll change it

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**